guides/deploy/README.md

```diff
@@ -144,12 +144,25 @@ Please note if you are running on the testnet, you are limited in the amount of
 
 {% endhint %}
 
+### Mainnet - Audited Attributes
+
+On the `akashnet-2` network, to ensure tenants have smooth and reliable service from their provider, it is recommended to use the following audited attributes in their deployment:
+
+```sh
+      attributes:
+        datacenter: equinix-metal-ams1
```

Comment:
:+1: 
The valid values for this are

* equinix-metal-ams1
* equinix-metal-ewr1
* equinix-metal-sjc1

The correspond directly to the Equinix Metal regions with the same name. We should advise people to choose one of them, based on their needs.